osu.Game/Overlays/Profile/Header/FullUserBindingComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Game.Users;
+
+namespace osu.Game.Overlays.Profile.Header
+{
+    public abstract class FullUserBindingComponent : UserBindingComponent
```

Comment:
what's a "fulluser"?

Reply:
As you can see, it's a Component which provides both User and Statistics bindables. And yes, I have some problems with naming